CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
 option(USE_GDBSTUB "Build Vita3K with GDB Debugger." OFF)
-option(USE_DISCORD_RICH_PRESENCE "Build Vita3K with Discord Rich Presence" OFF)
+option(USE_DISCORD_RICH_PRESENCE "Build Vita3K with Discord Rich Presence" ON)
```

Comment:
Ahh I get it. But now this option does nothing.

Reply:
no, this option is used, and tested, if exemple dev want compile release but disable discord rpc ^^ so by default is now on, but you can disable this option if you want :)

Reply:
You're right again! Damn.